### PR TITLE
Add Picross e8

### PR DESCRIPTION
**Pull request by gianveloxbr**
Adds Picross e8.

*Automatically generated by PR builder*

### CODE REVIEW

**Comment by gianveloxbr** [comment]
Done. Changed everything that was needed.

File: games/picross-e8/game.dat [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+title = "00040000001CF700"
+region = "usa"
+release_date = "2017-12-21"
+
+[[releases]]
+title = "00040000001CE000"
+region = "eur"
+release_date = "2017-12-21"
+
+[[ testcases ]]
```
Comment (Leo626) [inline games/picross-e8/game.dat]: Testcases aren't suppose to be here anymore. It is submitted through Citra now (Help -> Submit Compatibility).

File: games/picross-e8/game.dat [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+[[releases]]
+title = "00040000001CF700"
+region = "usa"
+release_date = "2017-12-21"
+
+[[releases]]
+title = "00040000001CE000"
+region = "eur"
+release_date = "2017-12-21"
```
Comment (Leo626) [inline games/picross-e8/game.dat]: Should be `2018-01-18`.

**Comment by gianveloxbr** [comment]
Updated as requested by Leo626.

**Comment by Leo626** [comment]
Add `jpn` release too `00040000001CFD00`. Same release date as `usa`.

**Comment by gianveloxbr** [comment]
Updated.

**Pull request merged**